src/components/scene.js

```diff
@@ -127,9 +141,19 @@ export class Scene extends Component {
         console.log("this.deckgl", prop);
       };
 
+      componentWillUnmount() { // @Chris Note that modal unmounts cleanly. This was the check I put in
+        console.log("Reached componentWillUnmount")
+      }
+      
+      // @Chris trying to avoid this force update. However, I'm unsure of what props need to be passed
```

Comment:
What’s the current need for it? This will just cause a react update after mounting, I’m not sure what benefit that has.

Reply:
This was one of two checks to narrow down the issues with the modal bug where you open it 2+ times and the TileLayer stops rendering. I wanted to make sure just the modal was unmounting properly